braintree/popup-bridge-ios
# Convert UI and Unit Tests to Swift

### Summary of changes

I started spiking out the changes to use `ASWebAuthenticationSession` in Popup Bridge. Since the tests were in Obj-C it would have required making things public that we don't want public in order to test them in Obj-C. With the tests in Swift it will make the switch to `ASWebAuthenticationSession` easier. All tests are converted 1:1 from Obj-C.

 - Convert `PopupBridge_DemoUITests` to Swift
 - Convert `PopupBridge_UnitTests`
     - Add `MockUserContentController` to replace `OCMock` which is Obj-C only
     - Add `MockScriptMessage` to replace `OCMock` which is Obj-C only
     - Add `MockPopupBridgeDelegate` to replace `OCMock` which is Obj-C only
 - Remove `OCMock` from `Podfile` and `Podfile.lock` - this pod is no longer in use

 ### Checklist


### Authors
> List GitHub usernames for everyone who contributed to this pull request.

- @jaxdesmarais
